frontend/components/site/SiteUsers.jsx

```diff
@@ -27,22 +38,31 @@ const SiteUsers = ({ site, user }) => {
         <thead>
           <tr>
             <th>User</th>
+            <th>Actions</th>
           </tr>
         </thead>
         <tbody>
-          {users.map(u =>
-            (<tr key={u.username}>
+          {users.map(rowUser =>
+            (<tr key={rowUser.username}>
               <td>
                 <a
-                  href={`https://github.com/${u.username}`}
+                  href={`https://github.com/${rowUser.username}`}
                   target="_blank"
                   rel="noopener noreferrer"
-                  title={`Visit GitHub profile for ${u.username}`}
+                  title={`Visit GitHub profile for ${rowUser.username}`}
                 >
-                  {u.username}
+                  {rowUser.username}
                 </a>
                 {' '}
-                {u.username.toLowerCase() === user.username.toLowerCase() ? '(you)' : ''}
+                {isSiteOwner(user, rowUser) ? '(you)' : ''}
+              </td>
+              <td>
+                {
+                  isSiteOwner(user, rowUser) ? '-' :
+                  <ButtonLink clickHandler={handleClick(rowUser)}>
+                    Remove user
```

Comment:
I've noticed that sometimes (but not always) after clicking this link that the table doesn't always re-rerender without the removed user. Have you noticed that? I haven't seen anything obvious that cause that behavior, but it definitely seems like something is up o_O

Reply:
Maybe something with the `SITE_USER_REMOVED` case in the reducer?

Reply:
I haven't noticed this, does anything look obviously wrong in the test in `test/frontend/reducers/sitesTest.js` (starting on line 231)?

Reply:
Is there a specific use case that triggers this, or does it seem to randomly happen?

Reply:
Seems to be random. 🤷‍♀️ 
The test looks correct to me.